tests/test_comparison_performance.py

```
@@ -0,0 +1,314 @@
+import sys
+import time
+from contextlib import contextmanager
+
+# Add the project root to the Python path
+project_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, project_root)
+
+from tests.utils.weac_reference_runner import (
+    compute_reference_model_results,
+    ensure_weac_reference_env,
+)
```

Ruff (0.12.2)

Module level import not at top of file

(E402)